sdk/react/controlBar.js

```diff
@@ -33,9 +33,12 @@ var ControlBarWidget = require('./widgets/controlBarWidgets');
 var CollapsingBarUtils = require('./collapsingBarUtils');
 var VolumeView = require('./widgets/VolumeView');
 var ResponsiveDesignManager = require('./responsiveDesignManager');
-
 var styles = Utils.getStyles(require('./style/controlBarStyles.json'));
 
+const constants = {
+  playbackSpeedRatePostfix: "x"
```

Comment:
This is probably fine but do some languages use a different symbol for multiplier? Should this be overridable?

Reply:
Disregard this. I asked JP and we don't allow you to customize it on web so you don't need to either.